ENOENT: Panicking when exec executables

Dunno why but `yofi binapps` tries to run $TERM when `execvp`.

```
> yofi --verbose binapps
[2022-06-30][13:45:10][yofi::exec][DEBUG] execvp: "tmux-256color" ["tmux-256color", "/home/crazybyte/.local/bin/weechat"]
thread 'main' panicked at 'execvp failed: ENOENT', src/exec.rs:60:41
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[2022-06-30][13:45:10][yofi][ERROR] panic: execvp failed: ENOENT
```

I saw an undocumented option to overwrite it, so I set `term` to `zsh` and it works.

~/.config/yofi/yofi.config:
```
[...]
term = "zsh"
[...]
```

Might I know why it is happening and if it is an intended behavior?

I'm using `yofi-git` on Archlinux.

--- a/src/config.rs
+++ b/src/config.rs
@@ -131,8 +131,6 @@ impl Config {
                 .into_iter()
                 .map(|s| CString::new(s).unwrap())
                 .collect::<Vec<_>>()
-        } else if let Ok(term) = std::env::var("TERM") {
-            vec![CString::new(term).unwrap()]
         } else {
             vec![]
         }

